No betas in Changelog (#2043)
RobinTail/express-zod-api: CHANGELOG.md

### v10.0.0

> - Fixed DTS path for ESM in package.json.

Author: ```suggestion
```

### v8.0.0

> - **Breaking**: removing the signature deprecated in v7.6.1.
>   - The argument of `EndpointsFactory::addMiddleware()` has to be the result of `createMiddleware()`;

Author: ```suggestion
- **Breaking changes**: 
  - Removed the signature deprecated in v7.6.1:
    - The argument of `EndpointsFactory::addMiddleware()` has to be the result of `createMiddleware()`;
```

### v5.1.0

- Feature #252: a helper method for testing your endpoints: `testEndpoint()`.
  - Requires `jest` (and optionally `@types/jest`) to be installed.
  - The method helps to mock the request, response, config and logger required to execute the endpoint.
  - The method executes the endpoint and returns the created mocks.
  - After that you only need to assert your expectations in the test.

> - Fixed a warning message when using `testEndpoint()` method.

Author: ```suggestion
```

### v5.0.0

- **Breaking changes**: Instead of HTTP Server the method `createServer()` now returns an object with the following
  entities: `app, httpServer, httpsServer, logger`.

> - The ability to configure and run an additional HTTPS server to process requests over a secure protocol.
> - This option is only available when using `createServer()` method.

Author: ```suggestion
- Feat: the ability to configure and run an additional HTTPS server to process requests over a secure protocol.
  - This option is only available when using `createServer()` method.
```

  complicated, but I found it important. However, it brought a number of benefits, which are also described below.
- Node version required: at least `10.0.0` and the library target now is `ES6`.
- Type `ResultHandler` is no longer exported, please use `createResultHandler()` or `defaultResultHandler`.
- The optional property `resultHandler` of `ConfigType` has been replaced with `errorHandler`.
- The `setResultHandler()` method of `EndpointsFactory` class has been removed. The `ResultHandlerDefinition` has
  to be specified as an argument of `EndpointsFactory` constructor. You can use `defaultResultHandler` or
  `createResultHandler()` for this, or you can use `defaultEndpointsFactory`.

> - Minor changes of response descriptions in OpenAPI / Swagger documentation generator.
> - The code has not been changed from the previous version.

Author: ```suggestion
```
